scripts/utils.js

```diff
@@ -176,6 +176,49 @@ async function updateDeploymentScriptVersion(nextVersion) {
   );
 }
 
+/**
+ * @param {string} importName
+ * @returns {[string, string]} [packageName, importPath]
+ */
+const getPackageNameFromImportName = (importName) => {
+  let [part1, part2, ...rest] = importName.split("/");
+
+  return importName.startsWith("@")
+    ? [`${part1}/${part2}`, rest.join("/")]
+    : [part1, [part2, ...rest].join("/")];
+};
```

Comment:
You coul use the `limit` parameter of `String.split` to make the logic easier to follow:

```suggestion
const getPackageNameFromImportName = (importName) => {
  if (importName.startsWith("@")) {
    let [ scope, pkg, path] = importName.split("/", 2)
    return { package: `${scope}/{pkg}`, path }
  }
  let [pkg, path] = importName.split("/", 1)
  return { package: pkg, path }
};
```

Reply:
@MichaelDeBoey ^ not sure if you saw my second comment. I'm pretty sure `split` with a limit doesn't return things after the limit unfortunately.

Reply:
Oops, I missed that comment 🙈

It doesn't seem to include the rest indeed
> [...] Any leftover text is not included in the array at all.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#limit

Will revert back to what I had.